### PR TITLE
Update docs on renaming existing package

### PR DESCRIPTION
**Pull request by ericphanson**
To mention the old package's URL should be updated. cc @oxinabox who wrote the original version

### CODE REVIEW

**Review by DilumAluthge — APPROVED** [review]

**Comment by DilumAluthge** [comment]
@ericphanson @oxinabox Is this good to merge?

**Comment by ericphanson** [comment]
I think so. Do you think the link is OK? That was the only PR I could find which actually did the thing I suggest to do here, but the main PR body has a lot of discussion which I thought could confuse people.

**Comment by DilumAluthge** [comment]
Hmmm. Maybe pick a simpler PR?

**Comment by DilumAluthge** [comment]
Or just omit the PR link. I agree that the discussion in that PR will probably be overwhelming and not terribly useful.

File: README.md [hunk 1]
```diff
@@ -126,10 +126,13 @@ As long as the package is not yet registered, renaming the package from
...
 
 Technically, you can't rename a package once registered, as this would break existing users.
-But you can re-register the package again under a new name with a new UUID.
-Which has basically the same effect.
+But you can re-register the package again under a new name with a new UUID, which basically
+has the same effect.
 
  - Follow the instructions above for renaming a package: rename on GitHub, rename files etc.
+    - if you rename the repository so it has a new URL, make a PR to edit the URL stored in the
+      registry for the old package name to point to the new URL ([example](https://github.com/JuliaRegistries/General/pull/11578/files)).
```
Comment (DilumAluthge) [inline README.md:134]:
```suggestion
      registry for the old package name to point to the new URL.
```

**Comment by ericphanson** [comment]
These are the repos whose URL in the registry redirects to another URL (ignoring `.git` ending differences:

|                                          original_repo |                                        final_repo |
|--------------------------------------------------------|---------------------------------------------------|
|         https://github.com/sunoru/RandomNumbers.jl.git |   https://github.com/JuliaRandom/RandomNumbers.jl |
|      https://github.com/christopher-dG/GitForge.jl.git |           https://github.com/JuliaWeb/GitForge.jl |
|           https://github.com/zekeriyasari/Jusdl.jl.git |         https://github.com/zekeriyasari/Causal.jl |
| https://github.com/kul-forbes/ProximalOperators.jl.git | https://github.com/kul-optec/ProximalOperators.jl |
|        https://github.com/logankilpatrick/Slack.jl.git |        https://github.com/JuliaLangSlack/Slack.jl |

as of a couple weeks ago when I ran
```julia
get_url_after_redirect(url) = readchomp(`curl -ILs -o /dev/null -w %\{url_effective\} $url`)

GitHub does not like this and will block you temporarily:
repo_redirects_df = let
	repo_redirects_df = DataFrame(:original_repo => unique(df.repo))
	repo_redirects_df.final_repo = asyncmap(get_url_after_redirect, repo_redirects_df.original_repo)
	repo_redirects_df
end

Arrow.write("repo_redirects.arrow", repo_redirects_df)
```
and got suspended from GitHub for a day for not rate-limiting when preparing my JuliaCon talk with Mosè about how many packages have docs/tests/ci/license/# lines of code/etc in General.

Of those, only one is a rename. So I will just make the PR to do the rename, and then link to that PR here 😄.

File: README.md (hunk 1 above)
Comment (ericphanson) [inline README.md:134]:
```suggestion
      registry for the old package name to point to the new URL ([example](https://github.com/JuliaRegistries/General/pull/40190/files)).
```

**Pull request merged**